libbeat/outputs/amqp/client.go

```diff
@@ -463,12 +464,18 @@ func (c *client) prepareEvent(codec codec.Codec, incoming eventTracker, now time
 	if err != nil {
 		return nil, fmt.Errorf("exchange select: %v", err)
 	}
+	if exchangeName == "" {
+		return nil, fmt.Errorf("exchange name is missing")
+	}
```

Comment:
A blank exchange name is valid under AMQP: it's the default (nameless) exchange. We might not use it at BC but it probably can't be an error for this generic AMQP pipeline.